src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -1040,6 +1040,55 @@ public enum KeyContainerPermissionFlags
         ChangeAcl = 8192,
         AllFlags = 13111,
     }
+    public sealed partial class MediaPermission : System.Security.CodeAccessPermission, System.Security.Permissions.IUnrestrictedPermission
+    {
+        public MediaPermission() { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionAudio permissionAudio, System.Security.Permissions.MediaPermissionVideo permissionVideo, System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionImage permissionImage) { }
+        public MediaPermission(System.Security.Permissions.MediaPermissionVideo permissionVideo) { }
+        public MediaPermission(System.Security.Permissions.PermissionState state) { }
+        public System.Security.Permissions.MediaPermissionAudio Audio { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionImage Image { get { throw null; } }
+        public System.Security.Permissions.MediaPermissionVideo Video { get { throw null; } }
+        public override System.Security.IPermission Copy() { throw null; }
+        public override void FromXml(System.Security.SecurityElement securityElement) { }
+        public override System.Security.IPermission Intersect(System.Security.IPermission target) { throw null; }
+        public override bool IsSubsetOf(System.Security.IPermission target) { throw null; }
+        public bool IsUnrestricted() { throw null; }
+        public override System.Security.SecurityElement ToXml() { throw null; }
+        public override System.Security.IPermission Union(System.Security.IPermission target) { throw null; }
+    }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Assembly | System.AttributeTargets.Class | System.AttributeTargets.Constructor | System.AttributeTargets.Method | System.AttributeTargets.Struct, AllowMultiple = true, Inherited = false)]
+    public sealed partial class MediaPermissionAttribute : System.Security.Permissions.CodeAccessSecurityAttribute
+    {
+        public MediaPermissionAttribute(System.Security.Permissions.SecurityAction action) : base(default(System.Security.Permissions.SecurityAction)) { }
```

Comment:
why would we pass `default(SecurityAction)` to `base` instead of the `action` param?